tests/lib/rules/no-invalid-regexp.js

```diff
@@ -26,11 +26,11 @@ ruleTester.run("no-invalid-regexp", rule, {
         { code: "new RegExp('.', 'y')", options: [{ allowConstructorFlags: ["y"] }]},
         { code: "new RegExp('.', 'u')", options: [{ allowConstructorFlags: ["U"] }]},
         { code: "new RegExp('.', 'yu')", options: [{ allowConstructorFlags: ["y", "u"] }]},
-        { code: "new RegExp('\/', 'yu')", options: [{ allowConstructorFlags: ["y", "u"] }]},
+        { code: "new RegExp('\/', 'yu')", options: [{ allowConstructorFlags: ["y", "u"] }]}, // eslint-disable-line no-useless-escape
```

Comment:
Both of them resolve to the same regexp literal in the end, so i'll change this to `'/'`.